client/request.go

```diff
@@ -130,7 +131,19 @@ func (cli *Client) sendClientRequest(ctx context.Context, method, path string, q
 		if len(body) == 0 {
 			return serverResp, fmt.Errorf("Error: request returned %s for API route and version %s, check if the server supports the requested API version", http.StatusText(serverResp.statusCode), req.URL)
 		}
-		return serverResp, fmt.Errorf("Error response from daemon: %s", bytes.TrimSpace(body))
+
+		var errorMessage string
+		if strings.Contains(resp.Header.Get("Content-Type"), "application/json") {
```

Comment:
Hmm. Seems like I'm wrong and the Engine actually just returns `application/json`. Not sure where I found `charset=utf-8`. Perhaps it was in some documentation somewhere.

I've updated to just check for the exact string `application/json`.